docs/src/main/asciidoc/config-your-quarkus-applications-using-a-yaml-file_howto.adoc

```diff
@@ -0,0 +1,407 @@
+////
+This document is maintained in the main Quarkus repository, and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="configure-a-yaml-application-properties-file"]
+= Configure a YAML application properties file
+include::_attributes.adoc[]
+:categories: core
+:summary: Use `application.yaml` instead of `application.properties`.
+
+Optionally, you can use a YAML-formatted file, `application.yaml`, instead of the default `application.properties` file to configure your application.
+
+link:https://en.wikipedia.org/wiki/YAML[YAML] is a popular format.
+For example, Kubernetes relies on YAML to write various resource descriptors.
+
+.Prerequisites
+
+* You have installed JDK 11 or later and set the `JAVA_HOME` environment variable to specify the location of the Java SDK.
+
+* You have installed Apache Maven {maven-version}.
+
+// include::modules/con_overview-quarkus-configuration.adoc[leveloffset=+1]
+[id="con_overview-quarkus-configuration"]
+== Overview
+
+Configuration options enable you to change the settings of your application in a single configuration file.
+
+{project-name} supports configuration profiles that you can use to group related properties and switch between profiles as required.
+
+By default, {project-name} reads properties from the `application.properties` file located in the `src/main/resources` directory.
+Instead, you can configure {project-name} to read properties from a YAML file.
+
+When you add the `quarkus-config-yaml` dependency to your project `pom.xml` file, you can configure and manage your application properties in the `application.yaml` file.
+
+For more information, see the xref:proc_enabling_yaml_configuration[Enabling YAML configuration].
+
+{project-name} supports YAML configuration files through the `SmallRye Config` implementation of Eclipse MicroProfile Config.
+
+You can use the link:https://microprofile.io/project/eclipse/microprofile-config[MicroProfile Config] specification from the Eclipse MicroProfile project to inject configuration properties into your application and access them using a method defined in your code.
+
+{project-name} can also read application properties from different origins, including the following sources:
+
+* The file system
+* A database
+* A Kubernetes or OpenShift `ConfigMap` or Secret object
+* Any source that a Java application can load
```

Comment:
I fail to see what this has to do with this particular guide?

Reply:
I agree. Plus, we don't want to portray SmallRye Config (or any other SmallRye project) as just an implementation of Eclipse MicroProfile. All of them are way beyond that, so we try to keep the references to MicroProfile to a minimum.

Reply:
Thanks. I agree. Removed.